cypress/integration/buy-shirt.spec.ts

```diff
@@ -1,19 +1,31 @@
+import {
+  MenuContentPage, ProductListPage, ShoppingCartPage,
+  LoginPage, AddressPage, ShippingPage, PaymentPage,
+} from "../page/index";
+
+const menuContentPage = new MenuContentPage();
+const productListPage = new ProductListPage();
+const shoppingCartPage = new ShoppingCartPage();
+const loginPage = new LoginPage();
+const addressPage = new AddressPage();
+const shippingPage = new ShippingPage();
+const paymentPage = new PaymentPage();
+
+const email = "aperdomobo@gmail.com";
+const pass = "WorkshopProtractor";
+
 describe("Buy a t-shirt", () => {
   it("then the t-shirt should be bought", () => {
-    cy.visit("http://automationpractice.com/");
-    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
-    cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click();
-    cy.get("[style*=\"display: block;\"] .button-container > a").click();
-    cy.get(".cart_navigation span").click();
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
-    cy.get("#SubmitLogin > span").click();
-    cy.get(".cart_navigation > .button > span").click();
-    cy.get("#cgv").click();
-    cy.get(".cart_navigation > .button > span").click();
-    cy.get(".bankwire").click();
-    cy.get("#cart_navigation > .button > span").click();
-    cy.get("#center_column > div > p > strong")
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+    productListPage.addTShirtToCart();
+    shoppingCartPage.processToCheckOut();
+    loginPage.login(email, pass);
+    addressPage.processToCheckOut();
+    shippingPage.chooseShippingOption();
+    paymentPage.payByBankWire();
+    paymentPage.confirmOrder();
+    cy.get(paymentPage.getConfirmationMessage())
```

Comment:
would be better if you have the cy.get within the getConfirmationMessage method. In that way we will see something like this:
paymentPage.getConfirmationMessage().should(.....)

Which is more clear to read :)